### ISSUE TITLE
version 1.0.208 on macOS arm64 breaks certain character presses on Norwegian keyboard

### ISSUE THREAD

**Issue by Binker** [Eugeny/tabby#9746]
Version: 1.0.208
Platform: macOS arm64 23.5.0
Plugins: search-in-browser, highlight
Frontend: xterm-webgl

On Norwegian Apple keyboards, the following chars '|' '['  ']' broke when upgrading to 1.0.208. 
Those chars are produced holding down "option" and pressing 7, 8 and 9 respectively. Producing third level chars by adding "shift" to the mix does nothing (for example, option+shift+8 normally produces '{' ).
Instead in zsh nothing happens, in bash pressing option+numbers produces "(arg: 1)" "(arg: 2)" "(arg: 3)" etc.

I downgraded back to ver. 1.0.207 which behaves normally.

**Comment by bh** [comment]
I can confirm this issue on Mac. It is impossible to insert an @ or a pipe or other special chars.

**Comment by mossicrue** [comment]
Same for Italian layout on mac: `option` + `ò` should print `@` but doesn't work

**Comment by amonod** [comment]
I can confirm that I also have the same problem when trying to type @ on an AZERTY keyboard with 1.0.208 on macOS arm64 (works fine on 1.0.207)

**Comment by fuhrmannb** [comment]
I have the same issue on a French AZERTY keyboard with characters like `|` (AltGr+6), `\`(AltGr+7), `#`(AltGr+3).
Downgrading to 1.0.207 is my current workaround.

**Comment by larrycai** [comment]
Same issue for Swedish Keyboard, even can't use `@`, back to 1.0.207 solves the issue

**Comment by bh** [comment]
Mac OS downgrade:
```
brew uninstall tabby
wget https://raw.githubusercontent.com/Homebrew/homebrew-cask/484891b6f9341536d5b0527bb5e465b130aecb1c/Casks/t/tabby.rb
brew install --cask tabby.rb
```

**Comment by lhassan-aitmouss** [comment]
I can confirm that this issue also affects French AZERTY keyboards on MacBook Pro M1 running macOS (Sonoma 14.5). 
I'm experiencing the same problem with Tabby version 1.0.209.

The following key combinations do not produce the expected characters:
- { : Option + (
- } : Option + )
- [ : Shift + Option + (
- ] : Shift + Option + )
- | : Shift + Option + L

These key combinations work correctly in other applications, including other Electron-based apps like Visual Studio Code. The problem seems to be specific to Tabby.

I've tried reinstalling Tabby and resetting its configuration, but the issue persists.

Regards,

**Comment by pesaventofilippo** [comment]
Same isssue on macOS with the Italian QWERTY keyboard, @#[]{} don't work, basically any simbol that requires using the Option key

**Comment by piersandro** [comment]
Same problem here from .208 version. Nano shows a "cannot bind M+' key" error
On .209 the error is different and if I press several times option + à (to white a #) on a terminal it type this:
<img width="355" alt="image" src="https://github.com/user-attachments/assets/df52770d-661d-4ab8-936d-12bdf07755e4">
Please solve soon, it is impossible to work :(

**Comment by bh** [comment]
> Please solve soon, it is impossible to work :(

Please please please ASAP. Just create a fix or downgrade to 1.0.207.

**Comment by piersandro** [comment]
> Please please please ASAP. Just create a fix or downgrade to 1.0.207.

I use Warp as a backup, but I have hundreds of profiles saved on Tabby. Downgrading is a solution of course.

**Comment by mavgit1** [comment]
I have the same problem with the current version. `Option + G`or other `Option + ...`combinations don't work. 

AMR Macbook with Tabby v. 1.0.209
